src/layouts/EditHomepage.jsx

```diff
@@ -1171,14 +1173,36 @@ const EditHomepage = ({ match }) => {
   return (
     <>
       {itemPendingForDelete.id && (
-        <DeleteWarningModal
-          onCancel={() => setItemPendingForDelete({ id: null, type: "" })}
-          onDelete={() => {
-            deleteHandler(itemPendingForDelete.id)
+        <GenericWarningModal
+          isOpen={isOpen}
+          onClose={() => {
             setItemPendingForDelete({ id: null, type: "" })
+            onClose()
           }}
-          type={itemPendingForDelete.type}
-        />
+          displayTitle={`Delete ${itemPendingForDelete.type} section`}
+          displayText={`Are you sure you want to delete ${itemPendingForDelete.type}?`}
+        >
+          <Button
+            variant="ghost"
```

Comment:
should this be `clear` instead?

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/a4e792aaffbb26913a0cc9a40493c17faf42a5d7